test/unit/plugins/lint-quotes-test.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+var generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'quotes',
+
+  good: [
+    {
+      config: true,
+      template: '<img alt=\'tomster\' src="tomster.png">'
+    }, {
+      config: 'double',
+      template: '<img alt="tomster" src="tomster.png">'
+    }, {
+      config: 'single',
+      template: '<img alt=\'tomster\' src=\'tomster.png\'>'
+    }, {
+      config: 'double',
+      template: '<img alt src="tomster.png">'
+    }, {
+      config: 'double',
+      template: '<img alt src={{url-tomster}}>'
+    }, {
+      config: 'double',
+      template: '<img alt src={{url "tomster"}}>'
+    }, {
+      config: 'double',
+      template: '<img alt src={{url img="tomster"}}>'
+    }, {
+      config: 'double',
+      template: '<img alt src={{url (inner-helper "tomster")}}>'
+    }, {
+      config: 'double',
+      template: '<img alt src={{url hash-param=(inner-helper "tomster")}}>'
+    }, {
+      config: 'double',
+      template: '<img alt src="{{url "tomster"}}">'
+    }, {
+      config: 'double',
+      template: '{{url "tomster"}}'
+    }
+  ],
+
+  bad: [
+    {
+      config: true,
+      template: '<img alt=tomster>',
+      result: {
+        message: 'Quotes: you should use quotes for attributes',
+        source: 'alt=tomster',
+        line: 1,
+        column: 5
+      }
+    }, {
+      config: 'double',
+      template: '<img alt=\'tomster\'>',
+      result: {
+        message: 'Quotes: you got single quotes when you set quotes style to be double quotes',
+        source: 'alt=\'tomster\'',
+        line: 1,
+        column: 5
+      }
+    }, {
+      config: 'single',
+      template: '<img alt="tomster">',
+      result: {
+        message: 'Quotes: you got double quotes when you set quotes style to be single quotes',
+        source: 'alt="tomster"',
+        line: 1,
+        column: 5
+      }
+    }, {
+      config: 'single',
+      template: '<img alt={{url "tomster"}}>',
```

Comment:
This is fine (and is required for things like `<div style={{someSafeString}}></div>`).  We should disambiguate between `ConcatStatement` (which is in quotes) and `MustacheStatement` (which is not in quotes).